circle.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+
+workflows:
+  version: 2
+  default:
+    jobs:
+      - primary
+
+jobs:
+  primary:
+    docker:
+      - image: mbgl/7d2403f42e:android-ndk-r16b
+    working_directory: ~/code
+    environment:
+      JVM_OPTS: -Xmx3200m
+      BUILDTYPE: Debug
+      IS_LOCAL_DEVELOPMENT: false
+    steps:
+      - checkout
+      - restore_cache:
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum  "MapboxAndroidDemo/build.gradle" }}
+      - run:
+          name: Download Dependencies
+          command: ./gradlew androidDependencies
+      - save_cache:
+          paths:
+            - ~/.gradle
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum  "MapboxAndroidDemo/build.gradle" }}
+      - run:
+          name: Log in to Google Cloud Platform
+          shell: /bin/bash -euo pipefail
+          command: |
+            echo "${GCLOUD_SERVICE_ACCOUNT_JSON}" > secret.json
+            gcloud auth activate-service-account --key-file secret.json --project mapbox-android-demo
+            rm secret.json
+      - run:
+          name: Export secrets for developer-config.xml
+          command: |
+            #!/bin/bash
+            echo "${MAPBOX_DEVELOPER_CONFIG}" > SharedCode/src/main/res/values/developer-config.xml
```

Comment:
We could add `MAPBOX_DEVELOPER_CONFIG` within the [`secretKeysSetup` task](https://github.com/mapbox/mapbox-android-demo/blob/master/SharedCode/gradle-config.gradle) and remove this step.